lib/rolemodel_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module RolemodelRails
-  VERSION = '0.9.0'
+  VERSION = '0.9.1'
```

Comment:
I would bump to 0.10 since this adds a whole new generator.
```suggestion
  VERSION = '0.10.0'
```

Reply:
That makes sense. Updated